backend/src/main/java/com/capstone/moneytree/controller/UserController.java

```diff
@@ -56,10 +62,11 @@ public UserController(UserService userService) {
      * @return The new updated user from the database
      */
     @PostMapping("/register-alpaca-key")
-    @ModelAttribute
     public ResponseEntity<User> registerAlpacaApiKey(@RequestBody User userWithKey) {
         String key = userWithKey.getAlpacaApiKey();
-        if (key.isEmpty() || key.isBlank()) {
+
+        // TODO: Refactor this into a validation class validateString method to validate strings & message class too
+        if (StringUtils.isEmpty(key) || StringUtils.isBlank(key)) {
```

Comment:
When using String utils, this: `if (StringUtils.isEmpty(key) || StringUtils.isBlank(key))` becomes this:
`if (StringUtils.isBlank(key))` within that check, both are validated by the library ("" or null). So you dont need to check for both and test coverage will be bigger. It is really minor, no need to update the PR for that.

Reply:
Done! I did the changes cause we need to improve our test coverage.